src/backend/src/controllers/design-review.controllers.ts

```diff
@@ -4,6 +4,15 @@ import { getCurrentUser } from '../utils/auth.utils';
 import DesignReviewService from '../services/design-review.services';
 
 export default class DesignReviewController {
+  static async getAllChangeRequests(_req: Request, res: Response, next: NextFunction) {
```

Comment:
I think this should be getAllDesginReviews

Reply:
haha oops